keyboards/handwired/scottokeebs/scotto36/readme.md

```diff
@@ -0,0 +1,25 @@
+# Scotto36
+
+![Scotto36](https://i.imgur.com/MCGv0ZHh.jpeg)
+
+A 36 key handwired ortholinear ergo keyboard. Featuring an OLED display with Bongo Cat. Case files available [here](https://github.com/joe-scotto/scottokeebs).
+
+-   Keyboard Maintainer: [Joe Scotto](https://github.com/joe-scotto)
+-   Hardware Supported: ATmega32U4
+-   Hardware Availability: [Amazon](https://amazon.com)
```

Comment:
```suggestion
*   Keyboard Maintainer: [Joe Scotto](https://github.com/joe-scotto)
*   Hardware Supported: ATmega32U4
*   Hardware Availability: [Amazon](https://amazon.com)
```

You can _proactively_ reformat the readme file in your other PRs according to the [prescribed template](https://github.com/qmk/qmk_firmware/blob/master/data/templates/keyboard/readme.md).

Reply:
@filterpaper Done, updated them to use the template.